isubit/iastate_theme
# Development 1.7.5

1. Add `is_front` and views classes to `<body>` https://github.com/isubit/iastate_theme/pull/192
2. Add Hero Top and Hero Bottom theme regions https://github.com/isubit/iastate_theme/pull/193
3. Add block type classes to blocks https://github.com/isubit/iastate_theme/pull/194

### To test block type classes and body classes

1. Make a Sites+ D9 site with this branch of the base theme
2. Enable Landing Page to get Cards
3. Structure > Blocks > Custom Block Library
5. Create and place different kinds of blocks in theme regions
6. Inspect the HTML and **confirm** that any content blocks have a `block-type_` class.
7. **Confirm** the `block-type_` class is not showing up on view blocks or system blocks.
8. Enable something like Projects or News and create some nodes.
9. Set both views and nodes as `front`
10. **Confirm** the correct CSS classes are showing up on `<body>`. For example if you set People Directory to front, I'd expect to see `is_front`, `view_people`, `view-display_people_directory`.

### To test new theme regions
Theme regions must be redeclared in the subtheme, so making a Sites+ site would not actually test the base theme changes.

1. Create a plain D9 site with this branch of the base theme
2. Structure -> Blocks
3. **Confirm** Hero Top and Hero Bottom theme regions are present.
4. Place blocks in these regions and confirm they show up.
